File: Scripts/server/file-input-output/c-sharp/file-input-output.cs
class ReadFromFile
{

    private static volatile int r = 1;

    public static int executeTask(int i) {

	string str = System.IO.File.ReadAllText(@"../input.txt");
	System.IO.File.WriteAllText(@"output.txt", str);
	return i + 1;
    }
	
    static void Main()
    {
	for ( int i = 0; i < 1000000; ++i ) {
		r = executeTask(i + r);
	}
    }
}

File: Scripts/server/file-input-output/java/FileInputOutput.java
import java.io.*;
public class FileInputOutput {
  
  private static volatile String line; 
 
  public static void main (String[] args) throws IOException {

    for ( int i = 0; i < 1000000; ++i) {

    BufferedReader br = new BufferedReader(new FileReader("../input.txt"));
    BufferedWriter bw = new BufferedWriter(new FileWriter("output.txt"));
    while ((line = br.readLine()) != null) {
      bw.write(line);
      bw.newLine();
    }
    br.close();
    bw.close();
 }
  }
}
